src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2348,22 +2348,8 @@ private static byte[] ConvertEnvVarsToByteArray(StringDictionary sd)
             return bytes;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Start-Process.UseNewEnvironment parameter should reset environment variables for child process
```powershell
Expected exactly '1;VssAdministrator', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Start-Process.Tests.ps1: line 188
188:             Get-Content -LiteralPath $outputFile | Should -BeExactly "1;$userName"
```